app/views/hyrax/admin/admin_sets/edit.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for(:body_attributes, 'data-turbolinks="false"'.html_safe) %>
```

Comment:
use `provide` rather than content for.  Additionally add a comment about why you are doing this (perhaps reference the issue)